packages/create-svelte/utils.js

```diff
@@ -49,3 +49,22 @@ export function copy(from, to, rename = identity) {
 export function dist(path) {
 	return fileURLToPath(new URL(`./dist/${path}`, import.meta.url).href);
 }
+
+// Thanks to https://github.com/zkochan/packages/tree/main/which-pm-runs for this code!
+export function getPackageManager() {
```

Comment:
we use snake_case for internal code and camelCase for external APIs
```suggestion
export function get_package_manager() {
```

Reply:
okay i didn't knew that. ill make changes quickly.

Reply:
I made the changes, so it should be okay. You may want to test though. I just used the web UI and didn't test